app/views/practices/form/implementation.html.erb

```diff
@@ -15,6 +18,9 @@
       <%= render partial: "shared/messages", locals: {small_text: false} %>
     <section class="usa-section padding-top-0 p0-bottom implementation">
     <h1 class="font-sans-2xl line-height-sans-205 margin-bottom-3">Implementation</h1>
+      <div class="margin-top-2 margin-bottom-2">
+        <%= render partial: "practices/form/practice_last_updated", locals: {small_text: true} %>
+      </div>
```

Comment:
We can use the class margin-y-2 here, since both the top and bottom margin sizes are identical.